config/standard/contact-summary.templated.js

```diff
@@ -21,7 +42,7 @@ var context = {
     gmp: isCoveredByUseCaseInLineage(lineage, 'gmp'),
   },
   treatment_program: getTreatmentProgram(),
-  enrollment_date: extras.getTreatmentEnrollmentDate(),
+  enrollment_date: getTreatmentEnrollmentDate(),
```

Comment:
since functions like this are only used once, you might consider just using `extras.getTreatmentEnrollmentDate` instead of creating the alias on line 30. But this works.